src/MenuScreen.cpp

```
@@ -110,9 +110,27 @@
     draw(renderer);
 }
 
-MenuScreen::MenuScreen(MenuItem** items) : items(items) {
-    while (items[itemCount] != nullptr) {
-        itemCount++;
+MenuScreen::MenuScreen(std::vector<MenuItem*> items) : items(items) {}
```

Cppcheck (2.10-2)

[performance] Function parameter 'items' should be passed by const reference.

(passedByValue)